examples/query.ts

```diff
@@ -1,32 +1,74 @@
 #!./node_modules/.bin/ts-node
```

Comment:
@bednar changes in this file seem to be accidentally committed during 1.16 release with dbd20a5cd8d35719b7b9cffaf75f68250132d39e , these are reverted herein ... so it now properly matches the write.js example.

Reply:
@sranka like you said, it was a mistake. Sorry for trouble

Reply:
ok, I am moving this out from this PR into master